src/tactic/ui/tools/pipeline_canvas_wdg.py

```diff
@@ -7711,7 +7729,7 @@ def execute(self):
 
             process_colors[process] = color
 
-        return process_colors
+        return process_colors, task_search_keys
```

Comment:
Better to use 

self.info

Reply:
self.info['process_colors'] = process_colors
self.info['task_search_keys'] = task_search_keys

# no return value